apps/andi/test/unit/andi_web/live/edit_live_view_test.exs

```diff
@@ -133,7 +133,8 @@ defmodule AndiWeb.EditLiveViewTest do
         [:email],
         ["foo@mail.com"],
         ["kevin@space.org"],
-        ["my@little.gov"]
+        ["my@little.gov"],
+        ["test-email@email.com"]
```

Comment:
I take it hyphens didn't match the existing regex?

Reply:
Correct, the regex we replaced it with is the regex that PODM's uses to validate emails